pkg/network/cache/fake/infocache.go

```diff
@@ -38,7 +38,7 @@ func (f *fakeInterfaceCacheFactory) CacheForVMI(vmi *v1.VirtualMachineInstance)
 	return f.vmiCacheStores[vmi.UID]
 }
 
-func (f *fakeInterfaceCacheFactory) CacheForPID(pid string) cache.DomainInterfaceStore {
+func (f *fakeInterfaceCacheFactory) CacheDomainInterfaceForPID(pid string) cache.DomainInterfaceStore {
```

Comment:
At search functions "By" is usually used also it would be nice to denote what PID we are talking about so I would propose also "Cache" can by  a verb so we can use find wo what about:

`FindCacheDomainByLauncherPID` ?

Reply:
That would be inconsistent w/ the other method names.